### PR TITLE
WRN-14392: Fixed skipped tests for RadioDecorator

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed skipped tests for RadioDecorator

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRB-14392

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by vladut-clapou-lgp — APPROVED** [review]
LGTM

File: packages/ui/RadioDecorator/tests/RadioDecorator-specs.js [hunk 1]
```diff
@@ -90,75 +94,81 @@ describe('RadioDecorator', () => {
...
-			<Controller>
-				<Component active />
+			<Controller ref={parentControllerRef}>
+				<Component active ref={parentDecoratorRef} />
 				<Component />
-				<Controller data-child-controller>
+				<Controller data-child-controller ref={childControllerRef}>
 					<Component active />
-					<Component />
+					<Component ref={childDecoratorRef} data-testid="childInactiveDecorator" />
```
Comment (adrian-cocoara-lgp) [inline packages/ui/RadioDecorator/tests/RadioDecorator-specs.js:159]: data-testid can be deleted

File: packages/ui/RadioDecorator/tests/RadioDecorator-specs.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
 import '@testing-library/jest-dom';
 import {render, screen} from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import {createRef} from 'react';
 
 import {RadioControllerDecorator, RadioDecorator} from '../RadioDecorator';
 
+const expectToBeActive = (controller, decorator) => {
```
Comment (MikyungKim) [inline packages/ui/RadioDecorator/tests/RadioDecorator-specs.js:8]: Let's move this to line 21 like before.

File: packages/ui/RadioDecorator/tests/RadioDecorator-specs.js [hunk 3]
```diff
@@ -13,51 +14,53 @@ describe('RadioDecorator', () => {
 
 	const Controller = RadioControllerDecorator('main');
 
+	const expectToBeActive = (controller, decorator) => {
+		expect(controller.active).toBe(decorator && decorator.handleDeactivate);
+	};
+
 	test('should be activated when its prop is true on mount', () => {
+		const controllerRef = createRef();
```
Comment (MikyungKim) [inline packages/ui/RadioDecorator/tests/RadioDecorator-specs.js:22]: Oh, why did you change these? Please don't make changes tests were working. :)
The comment on Jira was confined to the skipped test.

**Review by MikyungKim — APPROVED** [review]
:shipit:
